t/deploy.t

```diff
@@ -81,10 +81,6 @@ ok($dh->version_storage->database_version, 'DB deployed');
 is($dh->version_storage->database_version, $dh->schema_version, 'Schema at correct version');
 is($ret, 0, 'Expected return value (0) for no action needed');
 
-# check systemuser exists after upgrade
-my $user = $schema->resultset('Users')->find({username => 'system'});
-ok($user->id, 'system user does exists');
-
```

Comment:
TLDR; I'm fine with removing it

This test is now useless. I added it to catch the system user creation during migration (where indeed was a bug), but since then creating system user moved to deployment subroutine, this means even the first `$version - 1` schema deployment in this test creates a system user.

The presence of system user on deployed system is checked in [t/06-users.t](https://github.com/os-autoinst/openQA/blob/master/t/06-users.t#L40)